runtime/include/float128.h

```diff
@@ -12,7 +12,16 @@
 typedef __float128 float128_t;
 #else
 /* __float128 is not available on AArch64 or other generic targets;
-   on AArch64 at least, long double is 128 bits in size. */
+ * on AArch64 at least, long double is 128 bits in size.
+ *
+ * Fixme: On Windows long double size is 64 bits, same as double is.
```

Comment:
Could you use uppercase for the `FIXME`? Makes it easier to grep for.

Reply:
Sure, updating that.